engine/engine/src/wscript

```diff
@@ -134,7 +133,7 @@ def build(bld):
         protoc_includes = '../proto',
         target = 'dmengine',
         bundleid = 'com.defold.engine',
-        source = main_cpp.split(),
+        source = main_cpp.copy(),
```

Comment:
Call `copy` here to produce separate list because in `export_symbols` task we change the list and as result that changes shares with other tasks that use `main_cpp` list.